frame/contracts/src/wasm/runtime.rs

```diff
@@ -986,6 +998,7 @@ pub mod env {
 	///
 	/// NOTE: This is a implementation defined call and is NOT a part of the public API.
 	/// This call is supposed to be called only by instrumentation injected code.
+	/// It deals only with the *ref_time* Weight.
```

Comment:
maybe we rename amount -> ref_time ?

Reply:
let's do that in a separate PR, crate-wide. but for the added new host function functions I refactored the parameter names a bit.